src/swap/saga.ts

```diff
@@ -79,6 +94,10 @@ export function* swapSubmitSaga(action: PayloadAction<SwapInfo>) {
     ? fromToken.balance.shiftedBy(fromToken.decimals).toString()
     : ''
 
+  const estimatedUsdValue = fromToken
```

Comment:
What do you think about having 2 fields, one for the estimated usd value for the selling token, and another for the buying token?

Something like:
```
const estimatedUsdValueSellingToken = calculateEstimatedUsdValue({ tokenInfo: fromToken, tokenAmount: sellAmount })
const estimatedUsdValueBuyingToken = calculateEstimatedUsdValue({ tokenInfo: toToken, tokenAmount: buyAmount })
```

In theory, `fromToken`, `toToken`, `sellAmount` and `buyAmount` will be defined. So I don't know if we need to check them.
If they aren't defined we will have errors in other parts as well.

Reply:
:+1: